components/fraudlabs_pro/actions/screen-order/screen-order.mjs

```diff
@@ -0,0 +1,272 @@
+        ship_last_name: (typeof this.ship_last_name === "undefined") ? "" :`${this.ship_last_name}`,
+        ship_first_name: (typeof this.ship_first_name === "undefined") ? "" :`${this.ship_first_name}`,
+        ship_addr: (typeof this.ship_addr === "undefined") ? "" :`${this.ship_addr}`,
+        ship_city: (typeof this.ship_city === "undefined") ? "" :`${this.ship_city}`,
+        ship_state: (typeof this.ship_state === "undefined") ? "" :`${this.ship_state}`,
+        ship_country: (typeof this.ship_country === "undefined") ? "" :`${this.ship_country}`,
+        ship_zip_code: (typeof this.ship_zip_code === "undefined") ? "" :`${this.ship_zip_code}`,
+        email_domain: (typeof this.email_domain === "undefined") ? "" :`${this.email_domain}`,
+        user_phone: (typeof this.user_phone === "undefined") ? "" :`${this.user_phone}`,
+        email: (typeof this.email === "undefined") ? "" :`${this.email}`,
+        email_hash: (typeof this.email_hash === "undefined") ? "" :`${this.email_hash}`,
+        username_hash: (typeof this.username_hash === "undefined") ? "" :`${this.username_hash}`,
+        bin_no: (typeof this.bin_no === "undefined") ? "" :`${this.bin_no}`,
+        card_hash: (typeof this.card_hash === "undefined") ? "" :`${this.card_hash}`,
+        avs_result: (typeof this.avs_result === "undefined") ? "" :`${this.avs_result}`,
+        cvv_result: (typeof this.cvv_result === "undefined") ? "" :`${this.cvv_result}`,
+        user_order_id: (typeof this.user_order_id === "undefined") ? "" :`${this.user_order_id}`,
+        user_order_memo: (typeof this.user_order_memo === "undefined") ? "" :`${this.user_order_memo}`,
+        amount: (typeof this.amount === "undefined") ? "" :`${this.amount}`,
+        quantity: (typeof this.quantity === "undefined") ? "" :`${this.quantity}`,
+        currency: (typeof this.currency === "undefined") ? "" :`${this.currency}`,
+        department: (typeof this.department === "undefined") ? "" :`${this.department}`,
+        payment_mode: (typeof this.cvv_result === "undefined") ? "" :`${this.payment_mode}`,
+        flp_checksum: (typeof this.flp_checksum === "undefined") ? "" :`${this.flp_checksum}`,
+    };
+
+    const process1 = await new Promise((resolve, reject) => {
+                         flp.validate(params, (err, data) => {
+                                if (err) {
+                                    reject(err)  // calling `reject` will cause the promise to fail with or without the error passed as an argument
+                                    return        // and we don't want to go any further
+                                }
+                                resolve(data)
+                            })
+                        })
+                    .then(data => {
+                        //console.log(data);
+                        return (data);
+                    })
+                    .catch(err => {console.error(err)});
+    return process1;
```

Comment:
If you modify the file `fraudlabs_pro.app.mjs` as per my suggestion in the main comment, then you can do a refactor like this
```suggestion
    const {
      format,
      ip,
      lastName,
      firstName,
      billAddr,
      billCity,
      billState,
      billCountry,
      billZipCode,
      shipLastName,
      shipFirstName,
      shipAddr,
      shipCity,
      shipState,
      shipCountry,
      shipZipCode,
      emailDomain,
      userPhone,
      email,
      emailHash,
      usernameHash,
      binNo,
      cardHash,
      avsResult,
      cvvResult,
      userOrderId,
      userOrderMemo,
      amount,
      quantity,
      currency,
      department,
      paymentMode,
      flpChecksum,
    } = this;

    const response =
      await this.fraudlabsProApp.screenOrder({
        format: format ?? "json",
        ip,
        last_name: lastName ?? "",
        first_name: firstName ?? "",
        bill_addr: billAddr ?? "",
        bill_city: billCity ?? "",
        bill_state: billState ?? "",
        bill_country: billCountry ?? "",
        bill_zip_code: billZipCode ?? "",
        ship_last_name: shipLastName ?? "",
        ship_first_name: shipFirstName ?? "",
        ship_addr: shipAddr ?? "",
        ship_city: shipCity ?? "",
        ship_state: shipState ?? "",
        ship_country: shipCountry ?? "",
        ship_zip_code: shipZipCode ?? "",
        email_domain: emailDomain ?? "",
        user_phone: userPhone ?? "",
        email: email ?? "",
        email_hash: emailHash ?? "",
        username_hash: usernameHash ?? "",
        bin_no: binNo ?? "",
        card_hash: cardHash ?? "",
        avs_result: avsResult ?? "",
        cvv_result: cvvResult ?? "",
        user_order_id: userOrderId ?? "",
        user_order_memo: userOrderMemo ?? "",
        amount: amount ?? "",
        quantity: quantity ?? "",
        currency: currency ?? "",
        department: department ?? "",
        payment_mode: paymentMode ?? "",
        flp_checksum: flpChecksum ?? "",
      });

    $.export("$summary", "Successfully screen order");

    return response;
```